components/Chart.tsx

```diff
@@ -228,7 +228,7 @@ const Chart = (props: Props) => {
     .map((term: any) => ({
       pattern: _.find(patterns, ['_id', term.pattern?._ref]),
       patternName: _.find(patterns, ['_id', term.pattern?._ref])?.name,
-      type: _.capitalize(_.find(patterns, ['id', term.pattern?._ref])?.type) || term.rightsIntensity > 0 ? "Right" : "Obligation", // because pattern.type is not consistently populated
+      type: _.capitalize(_.find(patterns, ['_id', term.pattern?._ref])?.type),
```

Comment:
> Are there still many patterns without a type populated? (So we need a fallback based on intensity)

If we don't have this answer easily yet, maybe we keep the fallback until we turn on validation in Sanity? It's harmless right?

Reply:
So the fallback in this format doesn't actually work as expected - haven't worked out quite why (everything shows as a right if I leave it in). I suspect it will be JS being weird in comparing `undefined` to 0.

A query on Sanity (should have done this sooner!) shows that there aren't currently any patterns without a type defined though so I think we're ok here - 

![image](https://user-images.githubusercontent.com/20502206/216285688-faa43bc7-d541-4b84-aa79-d2f138689db2.png)

Reply:
great, thanks for double-checking, let's ditch it!